test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1410,8 +1410,12 @@ def revert_mom(self, mom, force=False):
         if not mom.isUp():
             self.logger.error('mom ' + mom.shortname + ' is down after revert')
         self.server.manager(MGR_CMD_CREATE, NODE, None, mom.shortname)
-        a = {'state': 'free', 'resources_available.ncpus': (GE, 1)}
+        a = {'state': 'free'}
         self.server.expect(NODE, a, id=mom.shortname, interval=1)
+        exp_attrib = {'resources_available.ncpus': (GE, 1)}
+        counter = self.server.counter(NODE, exp_attrib)
+        if not counter:
```

Comment:
counter() returns None or a dictionary, would it make more sense to check "if counter is not None".

Reply:
@prakashcv13: In case of failed condition, counter doesn't return None but a dictionary. Below is the snippet
Code-snippet:
        exp_attrib = {'resources_available.ncpus': (GE, 100)}
        counter = self.server.counter(NODE, exp_attrib)
        print("====>counter=", counter)

Log-snippet:
====>counter= {'resources_available.ncpus>=100': 0}

Reply:
hmmm, I am looking at the definition of counter in pbs_testlib.py - and see that it does return None. What you have tested is not an error condition.
In the function _filter() (called from counter()), if self.status raises PbsStatusError, counter will return None. There are other similar conditions as well in that function.